stories/core/ExpandableBlock.stories.tsx

```diff
@@ -111,6 +111,7 @@ export const Accordion: Story<Partial<ExpandableBlockProps>> = (args) => {
 Accordion.argTypes = {
   title: { control: { disable: true } },
   isExpanded: { control: { disable: true } },
+  isBorderless: { control: { disable: true } },
```

Comment:
`isExpanded` is set to `false` in all of the other stories' args except for `Small`. Should `isBorderless` do the same? Also, should both `isBorderless` and `isExpanded` be set to false in the `Small.args` as well?